Technicians

```diff
@@ -0,0 +1,12 @@
+--This is for the creation of the Technician table.
```

Comment:
No reason to put this comment in here.  It should be pretty apparent that's what this is for, plus we haven't put comments like this anywhere else.